src/js/view/view.js

```diff
@@ -409,6 +411,12 @@ define([
             this.checkResized();
         };
 
+        function itemReady() {
+            var title = _model.get('playlistItem').title || '';
+            var videotag = _videoLayer.querySelector('video, audio');
```

Comment:
@johnBartos title is still set. Checked when player does not appear on page and when item is loaded when player is in background tab